frontend/src/router.js

```diff
@@ -341,7 +381,7 @@ async function requireCamp(to, from, next) {
   await campFromRoute(to)
     .call({ api: { get: apiStore.get } })
     ._meta.load.then(() => {
-      next({ query: to.query })
+      next()
```

Comment:
why do we need this change?

Reply:
Because this line (blame) was non sensical to begin with and written by me 🫣